### PR TITLE
AEROGEAR-3059 document connecting your device to the service running locally on openshift

### PR DESCRIPTION
**Pull request by vchepeli**
## Motivation
To run with public IP we need to provide additional argument and pass public IP instead of default one

- [x] Read parameter for public IP
- [x] Pass parameter to ansible-playbook

## Additional Notes
[AEROGEAR-3059](https://issues.jboss.org/browse/AEROGEAR-3059)

### CODE REVIEW

File: docs/walkthroughs/connecting-device-to-service.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	inet 10.40.2.105 netmask 0xfffffe00 broadcast 10.40.3.255
+	inet 10.200.156.50 netmask 0xfffff800 broadcast 10.200.159.255
+```
+You have two IP address available for WiFi and ethernet, in this example WiFi network and 10.200.156.50 ip address is used.
+When installation is finished and successful you should get same result as on image below
+
+image::images/mobile-core-public-ip-success.png[openshift-mobile-core-public-ip-success]
+Web console URL should contain your public IP address, in this case it should be `10.200.156.50`
+
+[[mobile-core-export-self-signed-cert]]
```
Comment (maleck13) [inline docs/walkthroughs/connecting-device-to-service.adoc]: Do we want to link to the existing doc for this? https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/self-signed-certs.adoc

**Comment by maleck13** [comment]
May need a rebase

File: docs/walkthroughs/connecting-device-to-service.adoc (hunk 1 above)
Comment (PhilipGough) [reply docs/walkthroughs/connecting-device-to-service.adoc]: +1 we already have this doc and it negates the need for screenshots

**Comment by maleck13** [comment]
verified this works locally with @vchepeli via a call

**Pull request merged**